app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationHelper.java

```diff
@@ -516,17 +520,14 @@ public static void disablePullNotifications(Context context) {
 
     public static void clearNotificationsForActiveAccount(@NonNull Context context, @NonNull AccountManager accountManager) {
         AccountEntity account = accountManager.getActiveAccount();
-        if (account != null && !account.getActiveNotifications().equals("[]")) {
-            Single.fromCallable(() -> {
-                account.setActiveNotifications("[]");
-                accountManager.saveAccount(account);
-
-                NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
-                notificationManager.cancel((int) account.getId());
```

Comment:
Very odd indeed: This (old code) did work for me all the time.
It also did just now in the emulator: I had 19 notifications (lines) there when starting, switching to the notification tab removed them all.
Probably it always was a summary notification (so only one) and removing that one does ok.

Reply:
Possibly. I tend to bounce between a bunch of different branches, launching in the same emulator, so it wouldn't surprise me if the database and real life got out of sync.